packages/terra-modal-manager/tests/wdio/modal-manager-spec.js

```diff
@@ -313,10 +313,16 @@ Terra.describeViewports('ModalManager - Behaviors', ['large'], () => {
     });
 
     describe('Modal Focus', () => {
-      it('focuses on the modal when opened', () => {
+      it('is on the first element when modal when opened', () => {
         $('#root-component .disclose-small').click();
         $('[class*="slide-group"] #DemoContainer-1 .maximize').waitForDisplayed({ timeout: 1000 });
-        expect($('[aria-modal="true"][role="dialog"]').isFocused()).toEqual(true);
+
+        const firstElementWithModalBeginAttribute = $('[aria-modal="true"][role="dialog"] [data-terra-abstract-modal-begin="true"]');
+        expect(firstElementWithModalBeginAttribute.getText()).toBe('');
+
+        const expectedParentElement = $('[aria-modal="true"][role="dialog"]');
+        expect(firstElementWithModalBeginAttribute.parentElement().getHTML() === expectedParentElement.getHTML()).toEqual(true);
```

Comment:
Can this also be updated to:
```suggestion
        expect(firstElementWithModalBeginAttribute.parentElement().getHTML).toBe(expectedParentElement.getHTML());
```

Reply:
@sycombs & @sdadn - Good suggestion with the toBe() matcher. A new thing that we learned. 

This suggestion is missing parentheses. So, I made a manual commit.

Reply:
Oops, I missed a few in my suggestion. Thanks for doing the manual commit!